### PR TITLE
Add Node 6.2.1

### PR DESCRIPTION
**Pull request by variadico**
@yaso195 @tw4dl @zquestz @y0ssar1an

### CODE REVIEW

File: runtimes/node-6.2.1.conf [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+depends       [ { os: "ubuntu" } ]
+provides      [ { runtime: "node" },
+                { runtime: "node-6" },
+                { runtime: "node-6.2" },
+                { runtime: "node-6.2.1" } ]
+
+environment { "PATH": "/opt/apcera/node-6.2.1/bin:$PATH" }
+
+cleanup [ "/usr/lib/python2.7" ] # this will come from it using python
+                                 # it will generate *.pyc files
```
Comment (y0ssar1an) [inline runtimes/node-6.2.1.conf]: Node requires Python to build?

Comment (zquestz) [inline runtimes/node-6.2.1.conf]: Yep.

**Comment by tw4dl** [comment]
➜  runtimes git:(add-node-621) ✗ diff node-6.1.0.conf node-6.2.1.conf
< name:      "node-6.1.0"
---
> name:      "node-6.2.1"
5,6c5,6
<   { url: "https://s3.amazonaws.com/apcera-sources/node/node-v6.1.0-linux-x64.tar.gz",
<     sha256: "ce46dd0188181b70661b9162feffdbd8a860cb75cb6661c37d6d61982e3f72c5" },
---
>   { url: "https://s3.amazonaws.com/apcera-sources/node/node-v6.2.1-linux-x64.tar.gz",
>     sha256: "c6ae9c90858fb47a2915ad6494e5eb8e6f34f4512de1d5a461e5fb0c003590b1" },
13,14c13,14
<                 { runtime: "node-6.1" },
<                 { runtime: "node-6.1.0" } ]
---
>                 { runtime: "node-6.2" },
>                 { runtime: "node-6.2.1" } ]
16c16
< environment { "PATH": "/opt/apcera/node-6.1.0/bin:$PATH" }
---
> environment { "PATH": "/opt/apcera/node-6.2.1/bin:$PATH" }
22c22
<       export INSTALLPATH=/opt/apcera/node-6.1.0
---
>       export INSTALLPATH=/opt/apcera/node-6.2.1
27,28c27,28
<       tar xzvf node-v6.1.0-linux-x64.tar.gz
<       mv node-v6.1.0-linux-x64/* $INSTALLPATH
---
>       tar xzvf node-v6.2.1-linux-x64.tar.gz
>       mv node-v6.2.1-linux-x64/* $INSTALLPATH

**Comment by tw4dl** [comment]
LGTM

**Pull request merged**